p4-16/psa/PSA.mdk

```diff
@@ -293,15 +293,24 @@ few packets dropped for these reasons (perhaps only one).
 ## PSA type definitions {#sec-psa-type-definitions}
 
 Each PSA implementation will have specific bit widths for the
-following types in the data plane.  These widths should be defined in
-the target specific `psa.p4` include file.  They are expected to
-differ from one PSA implementation to another.
-
-For each one of these types, the P4 Runtime API uses values with the
-bit width of the corresponding `InHeader_t` type below to hold a value
-of that type, independent of the target.  All PSA implementations must
-use sizes for these types no wider than the corresponding types with
-`InHeader_t` in their names shown below.
+following types in the data plane.  These widths are defined in the
+target specific `psa.p4` include file.  They are expected to differ
+from one PSA implementation to another[^PSAP4TargetSpecific].
+
+[^PSAP4TargetSpecific]: It is expected that `psa.p4` include files for
+    different targets will typically be nearly identical to each
+    other.  Besides the possibility of differing bit widths for these
+    PSA types, the only expected differences between `psa.p4` files
+    for different targets would be annotations on externs, etc. that
+    the P4 compiler for that target needs to do its job.
+
+For each of these types, the P4 Runtime API[^P4RuntimeAPI] uses bit
```

Comment:
"may use" maybe?
I think the P4Runtime does say it is optional. In some cases maybe a controller does want direct access to the target-specific representation and numbering space.

Reply:
Sure.  The next commit will change "uses" to "may use" in that sentence.